libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.cpp

```diff
@@ -170,7 +171,7 @@ struct PACKED FIFODataHighRes {
 static_assert(sizeof(FIFOData) == 16, "FIFOData must be 16 bytes");
 static_assert(sizeof(FIFODataHighRes) == 20, "FIFODataHighRes must be 20 bytes");
 
-#define INV3_FIFO_BUFFER_LEN 8
+#define INV3_FIFO_BUFFER_LEN 16
```

Comment:
is this needed?

Reply:
Yes I believe it is

Reply:
.... needs explaining because this is definitely the sort of thing we end up needing a point release in stable for when we discover some random v3 sensors doesn't like the increased length...

Reply:
Added a comment